test/e2e/cases/19-verify-git-pull-time-metric/channel.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: Channel
+metadata:
+  name: gitops
+  namespace: git-pull-time-metric-test
+spec:
+  # change to https://github.com/open-cluster-management-io/multicloud-operators-subscription after merge
+  pathname: https://github.com/TomerFi/ocm-playground
```

Comment:
It would be better to define the channel by referring to our own multicloud-operator-subscription repo instead of personal github repos.

So the channel can be defined as 
```
spec:
  pathname: https://github.com/open-cluster-management-io/multicloud-operators-subscription.git
```

And put the git resources to the path `examples/git-simple-sub`

Reply:
@xiangjingli yes of course.
but I think this can only be done in another PR,
it won't work for this one because the manifests required are included in this PR and not in the _main_ branch yet.

the comment is my reminder, I will do it immediately after this PR get's merged.

Reply:
oops, noticed a typo in the example folder path - fixing and force pushing now.

Reply:
@TomerFi  Got it, Yeah, it is fine to open a new PR for changing the channel resources to our own subscription repo.

And Please also squash all commits into one.  Or the PR created in the midstream repo won't be handled correctly by our magic merge tool. The tool just handles the PR including one commit.

Reply:
@xiangjingli done.

Reply:
@xiangjingli channel spec set to own repo here #264